bindings/python/pymongocrypt/mongocrypt.py

```diff
@@ -242,6 +262,10 @@ def __raise_from_status(self):
             lib.mongocrypt_status_destroy(status)
         raise exc
 
+    @property
+    def csfle_version(self):
+        return lib.mongocrypt_csfle_version_string(self.__crypt, ffi.NULL)
```

Comment:
I'd suggest returning None instead of ffi.NULL when csfle is not loaded. The api does not otherwise expose the fact that it uses cffi.